### PR TITLE
code cleanup

### PR DESCRIPTION
**Pull request by flleeppyy**
## About The Pull Request

removes shitcode and calling things "retard" (to the pits of hell with ye)
also removes the uh. cum events? yeah.
archived changelogs contain lots of slurs, I didn't touch that for "archive" reasons but I can clean that up if needed.

## Why It's Good For The Game

we don't need a "limpdick" trait.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

File: code/datums/status_effects/rogue/debuff.dm [hunk 1]
```diff
@@ -87,34 +87,6 @@
 	desc = "My BANE!"
 	icon_state = "hunger3"
 
-
-// PINTLEDESTRUCTION
-
-/datum/status_effect/debuff/pintledestruction
-	id = "pintlegone"
```
Comment (dwasint) [inline code/datums/status_effects/rogue/debuff.dm:94]: why

Comment (dwasint) [reply code/datums/status_effects/rogue/debuff.dm:94]: pintle also adds a stress event not just limpdick

File: code/modules/admin/player_curses.dm [hunk 2]
```diff
@@ -4,9 +4,13 @@
 /proc/curse2trait(curse)
 	if(!curse)
 		return
-	switch(curse)
-		if("brokedick")
-			return TRAIT_LIMPDICK
+	// switch(curse)
+	// 	if("brokedick")
```
Comment (dwasint) [inline code/modules/admin/player_curses.dm:8]: just remove the code

File: code/modules/antagonists/roguetown/villain/zizocult.dm [hunk 3]
```diff
@@ -899,10 +899,6 @@ GLOBAL_LIST_EMPTY(ritualslist)
 		to_chat(H.mind, "<span class='notice'>I no longer feel pain, but it has come at a terrible cost.</span>")
 		H.change_stat("strength", -2)
 		H.change_stat("constitution", -2)
-		if(H.gender == FEMALE)
```
Comment (dwasint) [inline code/modules/antagonists/roguetown/villain/zizocult.dm:902]: make this just a -3 constitution

File: strings/accent_universal.json [hunk 4]
```diff
@@ -15,9 +15,6 @@
 		"humour": "humor",
 		"saviour": "savior",
 		"the earth": "creation",
-		"penis": "cock",
-		"retarded": "a natural born idiot",
```
Comment (dwasint) [inline strings/accent_universal.json:19]: leave the retarded ones in it converts anyone saying retard or retarded into those lines its better

Comment (flleeppyy) [reply strings/accent_universal.json:19]: Oh shit didnt realize, lol i just removed it cause i thought it wasn't meant to be there

File: code/datums/status_effects/rogue/debuff.dm (hunk 1 above)
Comment (flleeppyy) [reply code/datums/status_effects/rogue/debuff.dm:94]: why as in why did i remove it or like "why did this code exist"

Comment (dwasint) [reply code/datums/status_effects/rogue/debuff.dm:94]: why did you remove it outside of the limb dick it also gave a stress event

Comment (flleeppyy) [reply code/datums/status_effects/rogue/debuff.dm:94]: i removed it cause it made sense to remove, aberra wants to gut and replace rather than keep it.

Comment (dwasint) [reply code/datums/status_effects/rogue/debuff.dm:94]: but it doesn't replace the winebubble stress event in anyway just makes the winebubble worse

**Comment by KM-Catman** [comment]
jesus christ

**Pull request merged**